amr-wind/equation_systems/icns/source_terms/CoriolisForcing.cpp

```diff
@@ -83,9 +95,10 @@ void CoriolisForcing::operator()(
                                up[1] * vel(i, j, k, 1) +
                                up[2] * vel(i, j, k, 2);
 
-        const amrex::Real ae = +corfac * (un * sinphi - uu * cosphi);
+        const amrex::Real ae =
+            +(corfac * un * sinphi) - (corfac * uu * cosphi * S);
         const amrex::Real an = -corfac * ue * sinphi;
-        const amrex::Real au = +corfac * ue * cosphi;
+        const amrex::Real au = +corfac * ue * cosphi * S;
```

Comment:
S is a boolean why are you setting it equal to an amrex::Real ?

Reply:
Since a bool can be interpreted as 0 or 1, I was using S to remove or include those terms.